ldes-server-port-fetch/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fetchdomain/constants/RdfConstants.java

```diff
@@ -0,0 +1,29 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fetchdomain.constants;
+
+import org.apache.jena.rdf.model.Property;
+
+import static org.apache.jena.rdf.model.ResourceFactory.createProperty;
+
+public class RdfConstants {
```

Comment:
I really don't think we should be duplicating these constants. Sooner or later they will get out of sync and will result in bugs.
Do we really want to define some of these constants such as TREE in every module? That means that other modules can give it another String, which is something you would not expect when reading the code and can result in false assertions and again result in bugs.